ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.16.yaml

```diff
@@ -107,6 +107,16 @@ tests:
       ROTATE_CLOUD_CREDENTIALS: "true"
       TEST_SUITE: openshift/conformance/parallel
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-ovn-parallel-capi
+  minimum_interval: 24h
```

Comment:
isn't this too frequent? Every 2 or 3 days should be ok specially as is tech preview

Reply:
Why do you say "tech preview"? As far as I know, this is 4.16 GA material.

However, given that we already know it doesn't work yet, I am removing the periodic job and only leaving the presubmit. We may switch all our periodics to CAPI when we have a successful run.